docs/Tutorials/HelloWorld/NewProject.md

```diff
@@ -0,0 +1,80 @@
+# Getting Started: Creating an F´ Project 
+
+This tutorial will walk new users through creating a new F´ project.
+
+### Prerequisites:
+- [F´ Installation Requirements](../../INSTALL.md#requirements)
```

Comment:
This should probably be removed as a prerequisite here, since the installation is now done with the bootstrapping method explained below and we want to keep the barrier to entry low.